README.md

```diff
@@ -1,37 +1,133 @@
-- Hardware neutral: aims to be able to support existing and future 1st and 3rd party accelerator devices
+## Who should consider using NNFusion?
+- Developers who want to speed up the execution performance of their pre-defined or pre-trained DNN model.
+- Developers who want to deploy their pre-trained model as framework-free source codes with minimum library dependencies.
+- Researchers who want to quickly try new compiler optimization ideas or customize optimizations on some specific models.
 
-### Highlight features
-- Source-to-source compilation to remove framework overhead and dependency
-- Full stack of optimization passes built-in
-- Compile-time kernel co-scheduling abstraction for general accelerator devices
-- Tight integration and co-optimization with communication, data loading and job scheduling
-- Automatic kernel tuning through interacting with kernel tuning service
-- Compatible with both hand-crafted kernel code and vendor-provided libraries
-- Customized optimization support through directly rewriting the generated human-readable code
+## Highlight features
+- Provide a full-stack optimization mechanism, including:
+  - Data-flow graph optimizations, e.g., CSE, compile-time constant folding, etc.
+  - Model-specific kernel selection, kernel co-scheduling and kernel fusion
+  - Static memory layout and placement optimizations
+- Provide ahead-of-time and source-to-source (model-to-code) compilation to reduce runtime overhead and remove library/framework dependencies.
+- Support popular DNN model formats including TensorFlow and ONNX as input models
+- Support customized optimization in an easier and more efficient way, e.g., directly replacing hand-crafted kernels on the generated human-readable code.
+- Support commonly used devices like CUDA GPUs, and ROCm GPUs
 
-### Getting Started
-To try NNFusion, please read the tutorial on [Compile a Tensorflow model with NNFusion](https://github.com/microsoft/nnfusion/wiki/Compile-a-Tensorflow-model-with-NNFusion).
+## Get Started
+### Quick Start with Docker Image
+For end users, simply use docker to compile your model and generate high-performance executable.
 
-### Build and Test
-For more detailed building and testing information, please read the Build Guide.
+NNFusion supports and is well tested on Ubuntu 16.04 and 18.04 with a CUDA GPU equipped. 
 
+You should install nvidia-docker on your device to do the following steps.
+
+We will use a simple TensorFlow LSTM inference model as an example. You can download a frozen version from our model zoo:
+
+`wget https://nnfusion.blob.core.windows.net/models/tensorflow/frozen_lstm_l8s8h256_bs1.pb`
+
+To use your own model to get started, please refer to [Supported Models](https://github.com/microsoft/nnfusion/wiki/1.3-Supported-Models) to see whether it is supported and freeze your model according to [Freeze Your Model](https://github.com/microsoft/nnfusion/wiki/1.1-Freeze-TensorFlow-Models).
```

Comment:
can we point this supported model to the one @wenxiang shared on the meeting? that one has model repo links.

Reply:
[This page]( https://github.com/microsoft/nnfusion/blob/master/models/tensorflow/README.md) + [This page](https://github.com/microsoft/nnfusion/tree/osdi20_artifact/artifacts/models) = Supported models wiki page. Maybe we could add link to the wikipage instead of changing the readme.

Reply:
@wenxcs will we update (add new things in) these pages in the future? I'd prefer point the readme directly to the pages and get rid of the wiki page.

Reply:
Yes, we need to remove those in the future. By this motivation, I think changing the link to [this page](https://github.com/microsoft/nnfusion/blob/master/models/tensorflow/README.md) is great.

Reply:
If we want to remove wiki pages, I think we can change the link to [this page](https://github.com/microsoft/nnfusion/blob/master/models/tensorflow/README.md). Should I change it now?